lib/plugins/aws/invoke-local/index.js

```diff
@@ -810,7 +810,19 @@ class AwsInvokeLocal {
         this.options.extraServicePath || '',
         handlerPath
       );
-      const handlersContainer = require(pathToHandler);
+      let handlersContainer;
+      try {
+        handlersContainer = require(pathToHandler);
+      } catch (e) {
+        if (e.code === 'MODULE_NOT_FOUND') {
+          // Attempt to require handler with `.cjs` extension
+          pathToHandler = `${pathToHandler}.cjs`;
+          handlersContainer = require(pathToHandler);
```

Comment:
I would probably wrap it with `try/catch` and in case of `'MODULE_NOT_FOUND'` throw original error (othwerwise users will see that x.cjs module is not found when they use .js and it'll be confusing